packages/babel-plugin/src/__tests__/module-traversal.test.tsx

```diff
@@ -435,4 +449,109 @@ describe('module traversal', () => {
     expect(result).toHaveLength(1);
     expect(result[0]).toInclude('compiled/packages/babel-plugin/src/__fixtures__/mixins/simple.js');
   });
+
+  describe('Namespace imports', () => {
+    it('should replace a member expression referencing a default export', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ color: objects.default.primary }} />
+      `);
+
+      expect(result).toInclude('{color:blue}');
+    });
+
+    it('should replace a member expression referencing a named variable export', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ color: objects.colors.primary }} />
+      `);
+
+      expect(result).toInclude('{color:red}');
+    });
+
+    it('should replace a member expression referencing an export specifier', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ color: objects.danger }} />
+      `);
+
+      expect(result).toInclude('{color:blue}');
+    });
+
+    it('should inline css from an object spread', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ ...objects.style }} />
+      `);
+
+      expect(result).toInclude('{font-size:12px}');
+    });
+
+    it('should inline css from an call expression', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ ...objects.colorMixin() }} />
+      `);
+
+      expect(result).toIncludeMultiple(['{color:red}', '{background-color:pink}']);
+    });
+
+    it('should inline css from an identifier with an IIFE property', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ ...objects.fontMixin }} />
+      `);
+
+      expect(result).toInclude('{font-size:12px}');
+    });
+
+    it('should inline css from a function mixin with parameters', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        const color = { blue: 'blue' };
+
+        const Component = (props) => {
+          return <div css={{ ...objects.colorMixin2(color.blue) }} />
+        };
+    `);
+
+      expect(result).toIncludeMultiple([
+        '{color:red}',
+        // Cannot be resolved statically because of bug where
```

Comment:
We should probably make sure to link each workaround in the repo to a github issue in the comment so we can track what needs to be updated